modules/pgp/src/main/java/org/mule/module/pgp/DecryptStreamTransformer.java

```diff
@@ -70,7 +74,9 @@ public DecryptStreamTransformer(InputStream toBeDecrypted,
     public void initialize(OutputStream out) throws Exception
     {
         InputStream decodedInputStream = PGPUtil.getDecoderStream(this.toBeDecrypted);
-        PGPObjectFactory pgpF = new PGPObjectFactory(decodedInputStream);
+        BcKeyFingerprintCalculator calculator = new BcKeyFingerprintCalculator();
```

Comment:
The implementation of BcKeyFingerprintCalculator is stateless, so it may be instantiated a single time and reused.